decidim-core/app/cells/decidim/author_cell.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This cell renders the author of a resource. It is intended to be used
+  # below resource titles to indicate its authorship & such, and is intended
+  # for resources that have a single author.
+  class AuthorCell < Decidim::ViewModel
+    include LayoutHelper
+    include ::Devise::Controllers::Helpers
+    include ::Devise::Controllers::UrlHelpers
+    include Messaging::ConversationHelper
+
+    property :profile_path
+
+    def show
+      render
+    end
+
+    def profile
+      render
+    end
+
+    def profile_inline
+      render
+    end
+
+    private
+
+    def from_context
+      from_context = options[:from] if options[:from].present?
+      from_context = context[:from] if context[:from].present?
+      from_context
+    end
+
+    def from_context_path
+      resource_locator(from_context).path
+    end
+
+    def withdraw_path
+      from_context_path + "/withdraw"
+    end
+
+    def withdrawable?
```

Comment:
From this, it looks like an author (User/UserGroup) is withdrawable, commentable, etc. I'd expect an `AuthorCell` to render the author of a given resource (and only the author), not extra actions on the resource 😕 

Why not move it back to `AuthorBox`, or renaming it to `ResourceData` o something similar?

Reply:
Yes I understant what you mean, I can refactor it to decouple the actions to its own cell, `AuthorBox` :

```erb
<% if actionable? %>
  <div class="author-data__extra">
    <%= render :date %>

    <%= render :comments %>

    <%= render :flag %>

    <%= render :contact %>

    <%= render :withdraw %>
  </div>
<% end %>
```

```
decidim-core
└──app
    └── cells
      └── decidim
          ├── author
          │   ├── profile.erb
          │   ├── profile_inline.erb
          │   ├── show.erb
          ├── author_box
          │   ├── comments.erb
          │   ├── contact.erb
          │   ├── date.erb
          │   ├── flag.erb
          │   ├── show.erb
          │   └── withdraw.erb
          ├── author_box_cell.rb
          ├── author_cell.rb
          ...
```
But I'm not sure about naming it `AuthorBox` as it has many actions related to the context (`withdraw`, `flag`, `date` and `comments`) and only one to the author (`contact`).
So maybe naming it `AuthorContextActions` is more appropiate?